website/data/docs-nav-data.json

```
@@ -14,6 +14,10 @@
             "path": "intro/usecases",
             "hidden": true
           },
+          {
+            "title": "What is Service Discovery?",
```

```suggestion
            "title": "Service Discovery",
```

Can you make the other one (service mesh) similar? I think it fits better in the navbar like this